migrate2rocky/migrate2rocky.sh

```diff
@@ -265,6 +265,9 @@ repoinfo () {
     fi
     declare -gA repoinfo_results=()
     while IFS=" :" read -r name val; do
+	if [[ -z $name ]] && [[ -z $val ]]; then
```

Comment:
Slightly more efficient:
`if [[ -z $name && -z $val ]]; then`

...or could even do:
`if [[ ! ( $name || $val ) ]]; then`

Other than that I'm good with this.

I am slightly curious how that happened, though.  None of our other testing has had a problem with this, but it's a harmless change at worst so I'd say go ahead and pull it.

Reply:
Updated

> I am slightly curious how that happened, though. 

I'm just lucky like that.